CHANGELOG

```diff
@@ -1,3 +1,8 @@
+- 1.0.0.0
```

Comment:
I would prefer to bump the version to just `0.10.0.0`, regardless of what snap does.